assets/src/edit-story/components/panels/panel/panel.js

```diff
@@ -44,12 +45,34 @@ function Panel({
   ariaLabel = null,
   ariaHidden = false,
 }) {
-  const [isCollapsed, setIsCollapsed] = useState(false);
-  const [expandToHeight, setExpandToHeight] = useState(initialHeight);
-  const [height, setHeight] = useState(initialHeight);
+  const persisted = useMemo(() => {
+    const stored = localStorage.getItem(
+      `${LOCAL_STORAGE_PREFIX.PANEL}:${name}`
+    );
+    return stored && JSON.parse(stored);
```

Comment:
I think we need to wrap this in a try-catch to make it less fragile. Users can edit this themselves, random plugins can impact it too - stuff happens.